### ISSUE TITLE
Can you provide some general troubleshooting advice ?

### ISSUE THREAD

**Issue by adam-jones-net** [pdewacht/brlaser#181]
I have a 2320D and it worked fine at the moment of installing the driver.  However now a day or two later cups receives the jobs, the printer's _Ready_ light flashes for 10 or so seconds and then nothing, it just stops and no job prints.   

Cups seems none the wiser, in the web UI it just shows the job as complete.

In the _error_log_ file I see 1 line of possible interest?

> [Client 76] HTTP_STATE_WAITING Closing for error 32 (Broken pipe)

What is odd is that it seems to be able to print a windows test page fine, but as soon as I send it a light one page email to print it fails.

**Comment by QORTEC** [comment]
I have a similar issue with my `HL-3170CDW` printer, which doesn't use the `brlaser` driver. I've noticed that the printer works correctly after a fresh boot, but after an extended period of `uptime`, the Ready light starts blinking and the printer doesn't print any jobs.

I suspect that the issue is related to the high `uptime` on my computer, since I tend to keep it on or in sleep mode for several weeks before rebooting. As a workaround, I've found that rebooting my computer resolves the issue. I haven't had time to investigate the cause of the issue further, but I hope this helps.
